types/ComplianceEligibilityType.json

```diff
@@ -0,0 +1,19 @@
+{
+    "description": "Extends a hierarchical classification within a defined classification scheme for compliance or funding eligibility.",
+    
+    "allOf": [
+        {
+            "$ref": "../types/ClassificationType.json"
```

Comment:
I think its meaningful, but perhaps duplicative. For instance, it would be possible to represent `landCoverClass` and `eligibility` within the existing `classifications[]` introduced in `LandUseResourceType` (the "parent" class that `LandCoverResource` extends).

However, it's not readily discoverable - someone would need to know to work through the `classifications[]` array looking for these types. In particular, `landCoverClass` is a structured method of defining the type of land cover we are talking about (for instance, starting with forest and then indigenous forest), which is the key thing we need to know about LandCoverResources. So I think while there is some duplication, it is better to break it out for clarity.